### PR TITLE
Feat : Added Grid Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Grid Component in Component Library.
Screenshot (306)
Screenshot (307)

### CODE REVIEW

File: Components/Grid.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                <a href="/Components/Rating.html">
+                    <li class="sidebar__item">Rating</li>
+                </a>
+                <a href="/Components/Rating.html">
+                    <li class="sidebar__item">Grid</li>
+                </a>
+            </ul>
+        </div>
+        <!-- Grid -->
+        <!-- Grid Intro -->
```
Comment (Manju-sree) [inline Components/Grid.html:142]: any one comment is ok here

File: Component-assets/Css/component-Responsive.css [hunk 2]
```diff
@@ -59,4 +59,10 @@
   .sample__code{
     max-width:20rem;
   }
-}
+  .grid-two-container {
```
Comment (Manju-sree) [inline Component-assets/Css/component-Responsive.css:62]: good to see you are maintaining responsiveness of components

**Review by Manju-sree — APPROVED** [review]
code looks fine, but try to provide proper preview links and lit bit description about component and their variants, it's just a suggestion - try to add colors or images to grid . and there is a merge conflict in this PR. try to solve it before merging it.

File: Components/Grid.html (hunk 1 above)
Comment (SaurabhChawda) [reply Components/Grid.html:142]: > code looks fine, but try to provide proper preview links and lit bit description about component and their variants, it's just a suggestion - try to add colors or images to grid . and there is a merge conflict in this PR. try to solve it before merging it.

okay, I will keep in mind this suggestion, ThankYou

File: Component-assets/Css/component-Responsive.css (hunk 2 above)
Comment (SaurabhChawda) [reply Component-assets/Css/component-Responsive.css:62]: Thankyou

**Review by mariyasada — APPROVED** [review]
this looks fine to me. good to merge

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me.
well written and structured.
I think your UI is little bit dark in blue shade. you may check contrast ratio.
Because I got distracted, rather than components, background color is taking my attention.(personal opinion).
This PR is good to merge.

**Comment by SaurabhChawda** [comment]
> this looks fine to me. good to merge

ThankYou

**Comment by SaurabhChawda** [comment]
> This code looks fine to me. well written and structured. I think your UI is little bit dark in blue shade. you may check contrast ratio. Because I got distracted, rather than components, background color is taking my attention.(personal opinion). This PR is good to merge.

Okay, I will check the Contrast and Change the Background Color.
Thank you for the suggestion and Review.

**Pull request merged**